documentation/releaseNotes/releaseNotes.v7.1.1.md

```diff
@@ -0,0 +1,3 @@
+Today we are releasing the 7.1.1 build of the `dotnet monitor` tool. This release includes:
+
+- Represent System.Diagnostics.Metrics Counters as Gauges in Prometheus Exposition Format. ([#4101](https://github.com/dotnet/dotnet-monitor/pull/4101))
```

Comment:
The `release/7.1` branch was created from the `v7.1.0` tag and all non-feature related commits from `release/7.x` branch at the time since the tag were cherry-picked into the `release/7.1` branch; I did this because there were experimental feature commits that went into `release/7.x` while we were waiting for 7.1.0 to ship and they should not be included in `release/7.1`.